pkg/utils/utils.go

```diff
@@ -148,6 +150,18 @@ func TarDir(srcDir string, tarPath string) error {
 			}
 		}
 
+		for _, ex := range exclude {
+			if strings.Contains(path, ex) {
```

Comment:
Here, file format matching should be used, such as `*.log`, instead of using string containing functions.

Reply:
updated